docs/source/stability_policy.rst

```diff
@@ -0,0 +1,173 @@
+Here are a few hints on how these can be handled in a compatible way that is backward compatible or at least minimizes the impact of the change.
+Whether these hints are applied or the change is just implemented in a breaking way may be decided by the dev team on a case by case basis.
+This depends e.g. on the currently available capacity.
+
+1. Optional arguments become mandatory
+
+   1. Keep the argument optional in the method signature
+   2. If the argument is not passed, raise a ``ValueError`` with a helpful message
+
+   That way it's still a breaking change, but at least passed values are not randomly assigned to the wrong argument.
+
+2. New positional argument is added and changes the order of the arguments
+
+   1. Add it as optional argument
+   2. If the argument is not passed, raise a ``ValueError`` with a helpful message
+
+3. Optional argument is renamed
+
+   1. Keep the old argument name in the method signature
+   2. If the old argument is passed, issue a deprecation warning with a helpful message
+   3. If both the old and the new argument are passed, raise a ``ValueError`` with a helpful message.
+
+4. Mandatory argument is renamed
+
+   1. Keep the old argument name in the method signature and make it optional
+   2. If the old argument is passed, issue a deprecation warning with a helpful message
+   3. If neither the old nor the new argument is not passed, raise a ``ValueError`` with a helpful message
+   4. If the old argument is not the first mandatory argument, make the ones before optional as well and raise a ``ValueError`` if they are not passed
+
+When the Bot API changes attributes of classes, the method :meth:`telegram.TelegramObject.to_dict` will change as necessary to reflect these changes.
+In particular, attributes deprecated by Telegram will be removed from the returned dictionary.
+Deprecated attributes that are still passed by Telegram will be available in the :attr:`~telegram.TelegramObject.api_kwargs` dictionary as long as PTB can support that with feasible effort.
+Since attributes of the classes in the :mod:`telegram` package are not writable, we may change them to properties where appropriate.
+
+Development Versions
+~~~~~~~~~~~~~~~~~~~~
+
+Before a feature is in a release, i.e. the feature was merged into the ``master`` branch but not released yet, it is not covered by this policy and may change.
+Pre-Release marked as alpha, beta or release candidate are not covered by this policy either.
+
+Security
+~~~~~~~~
+
+We make exceptions from our stability policy for security.
+We will violate this policy as necessary in order to resolve a security issue or harden PTB against a possible attack.
+
+Versioning
+----------
+
+PTB uses a versioning scheme that roughly follows `https://semver.org/ <https://semver.org/>`_, although it may not be quite as strict.
+
+Given a version of PTB X.Y.Z,
+
+-  X indicates the major version number.
```

Comment:
add backticks around X, Y, Z?

Reply:
I don't quite see the need tbh … we don't put v20.2 in monospace either … 🤔